docs/concepts/portal/applications.md

```diff
@@ -0,0 +1,23 @@
+---
+title: " Applications Page Reference"
+linkTitle: "Applications Page Reference"
+---
+
+The __Applications__ tab displays all of your applications including the undeployed. For a detailed explanation of the view, refer to the table below:
```

Comment:
`including the undeployed`

Not exactly sure what you're referring to here. The applications tab shows an aggregation of information pertaining all statefulsets deployed across all the connected clusters - so I am a bit unclear to what this statement pertaining deployments refers to.

Maybe deployed as exposed?

Reply:
Do we even have such thing as not yet deployed apps. If not, I will just remove it.

Reply:
Not really, think the whole distinction should be removed